docs/tutorial/installation.md

```diff
@@ -11,14 +11,6 @@ npm install electron --save-dev
 See the [Electron versioning doc][versioning] for info on how to
 manage Electron versions in your apps.
 
-## Global Installation
```

Comment:
Rather than removing it all together we can consider changing it to:

## Running Electron without installing it

You can also run an electron without installing Electron:

```sh
npx electron .
```

This will let you run an electron app, but keep in mind any other dependencies will also not be installed.

Reply:
Updated!